src/fal/packages/dependency_analysis.py

```diff
@@ -60,3 +61,11 @@ def get_default_requirements() -> Iterator[Tuple[str, Optional[str]]]:
 
     for adapter_name, adapter_version in _get_dbt_plugins_info():
         yield f"dbt-{adapter_name}", adapter_version
+
+
+@cache_static
```

Comment:
ATL: what does this do?

Reply:
Think of it as a lazy global, until the function is called we don't construct anything and once there is at least one call we always return the same value (so pretty-much singleton-izing the thing).